src/store/messages/saga.ts

```diff
@@ -575,11 +566,11 @@ export function isOwner(currentUser, entityUserId) {
   return currentUser.id === entityUserId;
 }
 
-export function* sendBrowserNotification(channelId, message: Message) {
+export function* sendBrowserNotification(roomId, message) {
```

Comment:
Probably worth renaming `message` since that implies it's an actual chat message.